### ISSUE TITLE
Move packit-stg to MP+

### ISSUE THREAD

**Issue by jpopelka** [packit/deployment#470]
- [`packit--stg` project/namespace](https://console-openshift-console.apps.mpp-e1-preprod.syvu.p1.openshiftapps.com/k8s/cluster/projects/packit--stg)
- [Get started MP+ docs](https://source.redhat.com/groups/public/itpaas/managedplatformplushub/getstartedmpplus)

Pre-requisites:
- [x] #469
- [x] #471
- [x] #474
- [x] people in the Packit (Rover) group have `namespace-admin` role to be able to change RBAC
- [x] #490

Similar to #380, you should 
- update [vars/packit/stg_template.yaml](https://github.com/packit/deployment/blob/main/vars/packit/stg_template.yml) (and your local-only `stg.yaml`)
  - `project: packit--stg`
  - `host: https://api.mpp-e1-preprod.syvu.p1.openshiftapps.com:6443`
  - `postgres_version: 15` (to use [PostgreSQL-15](https://quay.io/repository/sclorg/postgresql-15-c9s) instead of default v13)
- [move the database](https://github.com/packit/deployment/blob/main/docs/postgresql-db-upgrade.md) first (`DEPLOYMENT=stg make deploy TAGS=postgres`)
- deploy the rest
- switch [DNS](https://domains.google.com/registrar/packit.dev/dns)
  - I'm not sure about the default router hostname, whether to take it from `status:ingress:routerCanonicalHostname` of a route or whether we need to consult [this doc](https://source.redhat.com/departments/it/digitalsolutionsdelivery/it-infrastructure/itcloudservices/itocp/managedplatformplushub/mppluswiki/how_to_manage_ingress_nad_egress_network_connectivity_for_mp)
- scale down the old one

**Comment by jpopelka** [comment]
My TODO (can be moved to a follow-up ticket):
- [ ] Use [internal Splunk image](https://source.redhat.com/departments/it/devit/it-infrastructure/itcloudservices/itocp/it_paas_kb/logging_to_splunk_on_managed_platform) because [our image](https://github.com/packit/fluent-plugin-splunk-hec)
  - [will reach End of Support on January 1, 2024](https://github.com/packit/fluent-plugin-splunk-hec#end-of-support)
  - tries to send the logs to [http-inputs-rhcorporate.splunkcloud.com](https://source.redhat.com/departments/it/splunk/splunk_wiki/splunk_fy20_architecture_blueprint#splunk-http-event-collectors-hec-) which is blocked by the firewall
- [ ] [Flower](https://github.com/mher/flower/blob/master/.github/workflows/docker.yml#L33) PR to [push the image also to Github Packages/Registry](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-docker-hub-and-github-packages)

**Comment by mfocko** [comment]
## Round 2

- [ ] Scale down auto-prod packit-stg
- [ ] Dump the postgres
- [ ] Deploy postgres
- [ ] Restore postgres
- [ ] Deploy the rest
- [ ] Check firewall
- [ ] Switch the DNS
- [ ] Scale down everything on the auto-prod
